### ISSUE TITLE
Theme customization for google amp

### ISSUE THREAD

**Issue by VanOkazou** [didomi/react#48]
Hello,

I am trying to add custom css (to `.didomi-button` for example), and, as specified in the documentation, the only way to override your inline css is to add `!important`.

Unfortunately for me, my css need to be compatible with Google Amp which does not allow `!important`.

Is there any way or may you add the possibility to remove your inline styles ? (globally or locally, for example only for `buttons.regularButtons`).

Thanks

**Comment by jawadst** [comment]
@VanOkazou Our CMP runs in its iframe and should not be subject to the usual AMP restrictions. You should be able to use `!important` in your custom CSS. Is there any reason why you cannot use `!important` in that context?

**Comment by VanOkazou** [comment]
@jawadst Thank you for your answer. You are right, `!important` works perfectly when I try to add css in your `theme.css` config.
Actually, I wanted to use my own app css file to override yours because I needed to use the css variables I have set in my root.
But I just realized that your `theme.css` was able to access to my css -variables.
I will close the issue, sorry for this inconvenience and thanks again.
